### DISCUSSION TITLE
SAI Injection Experiments (fixed SSTs)

### DISCUSSION THREAD

**Discussion by tilmes** [NCAR/CARMA_dev#26] [SAI Experiment]
@dan-visioni and Ilaria (could you add Ilaria's Github name)?
This is a discussion of the different runs we need for the SAI injection experiments with fixed SSTs.
We can list the general needs for the setup and all the runs needed here, e.g., control, MAM, CARMA, and then add the different runs in the I**ssues section.**

**Preparation before the actual runs and model evaluation:**
1. We discussed creating new 2deg emissions. I am currently working on this, since I have to remember what I did before I can share it, so more later.

2. We want to test the updated long-wave radiation in both CARMA and MAM4. I am suggesting first looking at one year of the ISAMIP runs,  to see if the radiation for the background conditions looks reasonable before we start a longer run.

I just started 1 year of a CARMA run, #Issue11 (very end). This run will include the new long-wave radiation variable "AODABSLW", which we want to compare to a new MAM4 run.
The output of the one-year run will be in /glade/scratch/tilmes/ISAMIP_CESM2CARMA_WACCMMA2000climo.carma_trop_stratv10.001 
and there will be more output when @zywshoon is running the longer run.

3. We have to then create an equivalent MAM4 run, that includes this variable.  To do this, we want to clone the MAM ISAMIP run from @dan-visioni  (as done in Issue#12), but use the latest code base (/glade/work/fvitt/camdev/carma_trop_strat10)  that should also include "AODABS10um" for MAM4.  We want to add this variable to the output to make sure it works. And then compare the output, and radiation variables. Hopefully they look more similar :)

After we finished the first part, we need to rerun this run with the new code-base and new output for CARMA: Issue#1 
We also want to add the "double radiation" call to the new run.

Note I already repeated the MAM4 run with the updated chemistry. However, I did not add the new long-wave output or the double radiation call. 
We probably want to re-run at least some period over Mt Pinatubo to add that output and the double radiation call, see Issue#25
After the evaluation, we can start the work on the actual control and injection runs.

Please add any comments/questions / suggestions below.

**Actual runs for the Injection study:**
@dan-visioni could you list all the runs we are planning to do and their setup? We can then discuss the details.
@ilaria-quaglia

**Comment by tilmes** [comment]
@dan-visioni @ilaria-quaglia while reading the CARMA paper again, I remembered that one of the findings are, that the pure sulfate mode in CARMA for the Mt Pinatubo case may not cover the largest particles and it may be interesting to check what happened if we added 2 more bins to the pure sulfate. We are also discussing changing the mixed aerosol range somewhat
I just want to share this, because, it may be of interest to some additional tests, with the current nudged version. Let me know if one of  you would be interested to try those at one point.

**Comment by dan-visioni** [comment]
Yes, this sounds really interesting!!
How do you modify the aerosol bins in CARMA?

**Reply by tilmes** [reply]
I think, it would make sense to explain the code a little bit and look at it.
We can do that during the next meeting or during a different meeting and maybe also add Ilaria to this?

**Comment by ilaria-quaglia** [comment]
Yes I would like to partecipate. Daniele and I are free to have a meeting do discuss this on Friday except from 1 to 2 pm EDT.

Best,
Ilaria

On 27 Mar 2023, at 14:42, tilmes ***@***.***> wrote:



I think, it would make sense to explain the code a little bit and look at it.
We can do that during the next meeting or during a different meeting and maybe also add Ilaria to this?
